microshift_getting_started/microshift-architecture.adoc

```diff
@@ -10,22 +11,54 @@ Learn the specifics of {product-title} architecture including design intent, how
+
+For example, {product-title} is tasked with only the following Kubernetes cluster capabilities:
+
+* Networking
+* Ingress
+* Storage
+* Helm
+
+{product-title} also provides the following Kubernetes functions:
+
+* Orchestration
+* Security
+
+{product-title} is optimized when used with a compatible operating system, such as {op-system-ostree-first}. When used together, {product-title} and {op-system-ostree-first} form {op-system-bundle}.
+
+.{product-title} as part of {op-system-bundle}.
+image::microshift-RHDE-Tech-Overview.png[<{product-title} is tasked with only the Kubernetes cluster services networking, ingress, storage, helm, with additional Kubernetes functions of orchestration and security.>]
+
+The following operational differences from {OCP} can help you understand where {product-title} can be deployed:
 
 [id="microshift-differences-ocp_{context}"]
 == Key differences from {OCP}
-* Application-level events and metrics are observation-only
+
 * Devices with {product-title} installed are self-managing
 * Compatible with RPM-OStree-based systems
 * Uses only the APIs needed for essential functions, such as security and runtime controls
 * Enables a subset of commands from the OpenShift CLI (`oc`) tool
+* Does not support workload high availability (HA) or horizontal scalability by the adding of worker nodes
+
+.{product-title} differences from {oke}.
+image::microshift-RHDE-v-OKE.png[<{product-title} is tasked with only the Kubernetes cluster capabilities of networking, ingress, storage, helm, with the additional Kubernetes functions of orchestration and security.>]
+
+Figure 2 shows that {oke} has the same cluster capabilities as {product-title}, and adds the following:
+
+* Install
+* Over-the-air updates
+* Cluster Operators
+* Operator Lifecycle Manager
+* Monitoring
+* Logging
+* Registry
+* Authorization
+* Console
+* Cloud Integration
+* Virtual Machines (VMs)
 
-// * You can use in locations where full control is needed
-// * Works offline and in low-connectivity environments
-// * You do not need to perform external orchestration
-// * Installs on top of {op-system-first}, an operating system especially for edge computing
-// * Use several single-node deployments to ... (use case example)
-// Bullets need more specific detail
+In {oke} and other {OCP} deployments, all of the components from the operating system through the cluster capabilities work as one comprehensive unit, with full cluster services for a multi-node Kubernetes workload. With {product-title}, functions such as over-the-air-updates, monitoring, and logging, are performed by the operating system.
 
 [id="microshift-openshift-apis_{context}"]
```

Comment:
I still think that his section on APIs does not belong to a "Getting stared Architecture" section. I would suggest to remove it here and move it for example to the CLI tools and API reference sections.

Reply:
I agree. I will move it when I set up that section--not a simple operation. The CLI tools and API reference sections are generated straight from the OCP docs, so I have to make a new book for this content in our docs. With the current set-up, I am not able to move this content there. I want customers to have the information, so this is sort of parked here for the moment. These are high-priority sections and are next on my list.